packages/gamut-labs/src/landingPage/PageFeatures.tsx

```diff
@@ -22,42 +32,47 @@ export type PageFeaturesProps = BaseProps & {
   maxCols?: 1 | 2 | 3 | 4;
 
   /**
-   * Array of features, which consist of image, image alt, title, and description
-   */
-  features: Omit<FeatureProps, 'featuresMedia' | 'onAnchorClick'>[];
-
-  /**
-   * @deprecated - Whether an icon or a full size image should be rendered
+   * The primary visual element.
    */
-  isIcon?: boolean;
+  featuresMedia?: 'image' | 'icon' | 'stat';
 
   /**
-   * The primary visual element.
+   * Array of features, which consist of image, image alt, title, and description
    */
-  featuresMedia?: 'image' | 'icon' | 'stat' | 'none';
+  features: {
+    title?: string;
+    desc?: string;
+    imgSrc?: string;
+    imgAlt?: string;
+    statText?: string;
+    testId?: string;
+  }[];
 };
 
 const rowRenderEach = (
-  items: FeatureProps[],
-  itemRenderer: (item: FeatureProps) => ReactNode
-): ReactNode => (
-  <FlexContainer nowrap column>
-    {items.map(itemRenderer)}
-  </FlexContainer>
-);
+  items: PageFeaturesProps['features'],
+  itemRenderer: (item: PageFeaturesProps['features'][0]) => ReactNode
+): ReactNode => {
+  /* eslint-disable react/no-array-index-key */
+  return (
+    <FlexContainer nowrap column>
+      {items.map((item, i) => (
+        <React.Fragment key={i}>{itemRenderer(item)}</React.Fragment>
+      ))}
+    </FlexContainer>
+  );
+  /* eslint-ensable react/no-array-index-key */
+};
 
 const gridRenderEach = (
   maxCols: NonNullable<PageFeaturesProps['maxCols']>,
-  items: FeatureProps[],
-  itemRenderer: (item: FeatureProps) => ReactNode
+  items: PageFeaturesProps['features'],
+  itemRenderer: (item: PageFeaturesProps['features'][0]) => ReactNode
 ): ReactNode => {
   const size = { xs: 12, sm: 12 / maxCols } as ResponsiveProperty<ColumnSizes>;
   /* eslint-disable react/no-array-index-key */
   return (
-    <LayoutGrid
-      columnGap={{ lg: 'lg', xs: 'sm' }}
-      rowGap={{ lg: 'lg', xs: 'sm' }}
-    >
+    <LayoutGrid columnGap={{ lg: 'lg', xs: 'sm' }} rowGap={0}>
```

Comment:
i _think_ we can just remove `rowGap={0}` here
```suggestion
    <LayoutGrid columnGap={{ lg: 'lg', xs: 'sm' }}>
```

Reply:
The compiler also complains because `0` isn't a valid value for this prop.
```tsx
Type 'number' is not assignable to type '"sm" |
 "md" | "lg" | "xl" | { xs: GapSizes; sm?: "sm" "md"
 | "lg" | "xl" | undefined; md?: "sm" | "md" | "lg" | 
 "xl" | undefined; lg?: "sm" | "md" | "lg" | "xl" | 
 undefined; xl?: "sm" | ... 3 more ... | undefined; } 
 | undefined'.ts(2322)
```

https://60143deb237e6e66885b982b--gamut-preview.netlify.app/storybook/?path=/docs/layouts-layoutgrid--grid-gaps

Reply:
fixed